### DISCUSSION TITLE
contributors page in ghostty.org

### DISCUSSION THREAD

**Discussion by ronit18** [ghostty-org/ghostty#3653] [Feature Requests, Ideas]
We should make a contributors page on website which will fetch contributors from git repo and show there icons and it will point to https://github.com/ghostty-org/ghostty/graphs/contributors

**Comment by 00-kat** [comment]
Wrong repo, you're looking for https://github.com/ghostty-org/website.

**Reply by ronit18** [reply]
Yep. Ik but there is no discussion on that repo. So i thought i could ask here.

**Reply by 00-kat** [reply]
Ah; just opened https://github.com/ghostty-org/website/issues/198; also consider asking in https://discord.com/channels/1005603569187160125/1319399404758761565 if you're in the Discord.

**Reply by 00-kat** [reply]
Apologies, I didn't realise [feature requests for the website also live here](https://github.com/ghostty-org/website/blob/main/CONTRIBUTING.md).
